repository/src/main/java/org/pentaho/platform/repository/hibernate/usertypes/BlobUserType.java

```diff
@@ -29,10 +29,15 @@
 import org.apache.commons.logging.Log;
 import org.apache.commons.logging.LogFactory;
 import org.hibernate.HibernateException;
+import org.hibernate.engine.spi.SharedSessionContractImplementor;
 import org.hibernate.type.SerializationException;
 import org.hibernate.usertype.UserType;
-import org.hibernate.util.EqualsHelper;
-import org.hibernate.util.SerializationHelper;
+//EqualsHelper removed after hibernate-core-5.3.1.Final.jar
+//import org.hibernate.util.EqualsHelper;
+//This SerializationHelper was moved from:
```

Comment:
Do we really need to keep these comments here?

Reply:
Thought I'd keep this one in, in case something crops up